packages/back-end/src/services/files.ts

```diff
@@ -79,9 +100,18 @@ export async function getFileUploadURL(ext: string, pathPrefix: string) {
       uploadURL,
       fileURL: S3_DOMAIN + (S3_DOMAIN.endsWith("/") ? "" : "/") + filePath,
     };
-  }
-  // Otherwise, use the local file system
-  else {
+  } else if (UPLOAD_METHOD === "google-cloud") {
+    const uploadURL = await getSignedGoogleUrl().catch(console.error);
+
+    if (!uploadURL) {
```

Comment:
@jdorn Let me know if you think this is the best location for throwing this error.

Reply:
Instead of this check, you can just get rid of `.catch(console.error)` above.  That way, if it fails to generate a URL, the `await` will cause it to throw an exception.  Right now, you are catching an exception and then throwing a new one later, so you can simplify by just not catching the first one.

Reply:
10-4. I totally meant to remove the .catch(console.error) initially. But, I see your point, I'll remove this and the check below.